Tests/ASDisplayNodeTests.mm

```diff
@@ -2266,12 +2266,10 @@ - (void)testThatItIsAllowedToRetrieveDebugDescriptionIncludingVCOffMainThread
   UIViewController *vc = [[UIViewController alloc] init];
   [vc.view addSubnode:node];
   dispatch_group_t g = dispatch_group_create();
-  dispatch_group_enter(g);
   __block NSString *debugDescription;
-  [NSThread detachNewThreadWithBlock:^{
+  dispatch_group_async(g, dispatch_get_global_queue(QOS_CLASS_DEFAULT, 0), ^{
     debugDescription = [node debugDescription];
-    dispatch_group_leave(g);
-  }];
+  });
```

Comment:
`detachNewThreadWithBlock:` is iOS10-only. I don't suspect we'll run our tests against iOS 8/9 too too much, but this was the only divergence so let's keep the option open.